src/controller.js

```diff
@@ -73,8 +88,8 @@ function buildSESKernel(initialState) {
   const r = s.makeRequire({
     '@agoric/evaluate': {
       attenuatorSource: `${makeEvaluate}`,
-      confineExpr: s.global.SES.confineExpr,
-      confine: s.global.SES.confine,
+      sesOptions: evaluateOptions,
```

Comment:
Any particular reason to not just name the property `evaluateOptions` instead of `sesOptions`? (one that comes to mind is the potential confusion, since an instance of "`evaluateOptions`" in `makeRequire` might appear to be shadowing the top-level name, but since `makeRequire` is only used for its source code, it's not really a shadowing)

Reply:
I called them `rootOptions` to indicate that they are used by the root realm.